src/Components/dotnet-runtime-js/dotnet.d.ts

```diff
@@ -3,7 +3,7 @@
 //!
 //! This is generated file, see src/mono/wasm/runtime/rollup.config.js
 
-//! This is not considered public API with backward compatibility guarantees. 
+//! This is not considered public API with backward compatibility guarantees.
```

Comment:
it seems your VS formatted the document ? Let's keep is binary same as the generated file.